types/karma-webpack/index.d.ts

```diff
@@ -0,0 +1,38 @@
+// Type definitions for karma-webpack 2.0
+// Project: http://github.com/webpack/karma-webpack
+// Definitions by: Matt Traynham <https://github.com/mtraynham>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import * as m from 'karma';
+import webpack = require('webpack');
+import webpackDevMiddleware = require('webpack-dev-middleware');
+
+declare module 'karma' {
```

Comment:
It seems not right to declare module `karma` in `karma-webpack`

Reply:
I kind of agree that it seems a bit weird, but this is basically an extension of the karma, or rather a module plugin. It appends two new properties to the Karma options.

It was started with the template `dts-gen --dt --name karma-webpack --template module-plugin`, which generates a [similar template](https://github.com/Microsoft/dts-gen/blob/master/templates/module-plugin.d.ts#L2).  It first imports the entire library you are adding to, and then declares a merged module to add new properties to.